book/chapters/gridmet.ipynb

```diff
@@ -22,7 +22,7 @@
   },
   {
```

Comment:
<img width="696" alt="image" src="https://github.com/geo-smart/swe-workflow-book/assets/45752727/8453446d-8aae-4ce7-8755-b66b1e882c9a">

can you change this to "Changing the timeframe from 10 years to 1 year to demonstrate"
currently it is reverse.

Reply:
done